reporter/src/web-app/js/src/polyfills.js

```diff
@@ -0,0 +1,55 @@
+// From: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Polyfill
```

Comment:
Not sure if we should use PolyFill or introduce something like Babel to translate ES6 to whatever IE support. Is IE support a must have for you

Personally I am developing the WebApp code without keeping IE in mind, want to be able to use all ES6 features that are supported by modern browsers

Reply:
Babel itself doesn't do polyfills, but babel-polyfill can be used. I don't personally much care about IE, but in my experience people working in enterprise settings use IE much more than the average person. I'll take the polyfill out for now.